Hymns/DetailHymnScreen.swift

```diff
@@ -34,14 +35,33 @@ struct DetailHymnScreen: View {
                     }.frame(width: geometry.size.width/1)
                 }.aspectRatio(contentMode: .fit).frame(height: 60).background(Color.white.shadow(radius: 2))
                 Spacer()
+                Spacer()
             }.frame(minHeight: 0, maxHeight: 60)
+            HymnLyricsView(viewModel: self.viewModel)
         }
     }
+
+    init(viewModel: HymnLyricsViewModel) {
+        self.viewModel = viewModel
+    }
 }
 
 struct DetailHymnScreen_Previews: PreviewProvider {
 
     static var previews: some View {
-        DetailHymnScreen()
+        //DetailHymnScreen()
+
+        let classic1151 = HymnLyricsViewModel(hymnToDisplay: PreviewHymnIdentifiers.hymn1151, hymnsRepository: Resolver.resolve(), mainQueue: Resolver.resolve(name: "main"))
+        classic1151.lyrics = classic1151_preview.lyrics
+        let classic1151View = HymnLyricsView(viewModel: classic1151)
```

Comment:
I think these need to be DetailHymnScreens

Reply:
addressed in recent commit